src/main/java/ca/warp7/frc2022/auton/commands/AutoLauncherCommand.java

```diff
@@ -0,0 +1,21 @@
+package ca.warp7.frc2022.auton.commands;
+
+import ca.warp7.frc2022.subsystems.Launcher;
+import ca.warp7.frc2022.subsystems.LauncherInterface;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class AutoLauncherCommand extends CommandBase{
+    private LauncherInterface launcher = Launcher.getInstance();
+    private boolean isEnabled;
+
+    public AutoLauncherCommand(boolean enabled) {
```

Comment:
I think that enabled is always true...  but I haven't gone all the way through this yet.  And that would kind of make sense,  we're using the launcher in autonomous mode so we turn it on.